Makefile

```diff
@@ -711,6 +716,7 @@ $(OBJDIR)/%/build/expat/configure:
 	rm -rf $(dir $@)
 	mkdir -p $(dir $@)
 	cp -a $(SRC_EXPAT)/* $(dir $@)
+	mkdir -p $(dir $@)/m4
```

Comment:
Why is this needed?

Reply:
It's obviously not needed for picolibc stuff, but when I tried to build locally, I got caught by a missing directory here. Probably just related to my local system configuration, but I didn't want to push anything to you that I hadn't tested.